sensors/baro/lps25xx.c

```diff
@@ -58,8 +59,10 @@
 #define SENSOR_CONV_KELV    0.002083333F /*conversion from sensor value to kelvin */
 #define SENSOR_KELV_OFFS    315.65F      /* temperature measurement offset + kelvin to celsius offset */
 
+
 typedef struct {
 	spimsg_ctx_t spiCtx;
+	oid_t *spiSS;
```

Comment:
Hmmm, I don't get it, why we want this to be a pointer to oid instead of plain oid?

Reply:
The idea is that `spiSS == NULL` means that chip select is handled by the controller. Otherwise `spiSS` stores GPIO pin oid which controls the chip select.